src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -122,24 +120,24 @@ export const pageContent = {
     ),
   },
   [AcknowledgementIdsEnum.slashing]: {
-    title: <FormattedMessage defaultMessage="Bad validator behaviour" />,
+    title: <FormattedMessage defaultMessage="Bad validator behavior" />,
     content: (
       <>
         <Text size="medium" className="my10">
           <FormattedMessage
             defaultMessage="If you try to cheat the system, or act contrary to the specification,
-              you will be liable to incur a penalty, known as slashing. Running your validator keys simultaneously on two or more machines will result in slashing."
+              you will be liable to incur a penalty known as slashing*. Running your validator keys simultaneously on two or more machines will result in slashing."
           />
         </Text>
         <Text size="medium" className="my10">
-          <FormattedMessage defaultMessage="Simply being offline with an otherwise healthy network does not result in slashing, but will result in small inactivity penalties." />
+          <FormattedMessage defaultMessage="*Simply being offline with an otherwise healthy network does not result in slashing, but will result in small inactivity penalties." />
```

Comment:
Just a random thought @wackerow 

Would be cool to have some sort of visual comparison of the slashing vs inactivity somewhere. People are starting to understand that offline !== slashed but I bet they still don't have a good understanding as to what each scenario means concretely for a validator balance